packages/admin-next/dashboard/src/hooks/table/filters/use-product-table-filters.tsx

```diff
@@ -144,21 +168,21 @@ export const useProductTableFilters = (
   //   filters = [...filters, collectionFilter]
   // }
 
-  const giftCardFilter: Filter = {
-    key: "is_giftcard",
-    label: t("fields.giftCard"),
-    type: "select",
-    options: [
-      {
-        label: t("fields.true"),
-        value: "true",
-      },
-      {
-        label: t("fields.false"),
-        value: "false",
-      },
-    ],
-  }
+  // const giftCardFilter: Filter = {
```

Comment:
nit: Can probably just remove this. I imagine the gift card will no longer be modeled using the product